sycl/test-e2e/ACPPCustomOperation/custom-command-hip.cpp

```diff
@@ -0,0 +1,138 @@
+// FIXME: the rocm include path and link path are highly platform dependent,
+// we should set this with some variable instead.
```

Comment:
This is FIXME that was intended to be resolved for this PR? Or a follow on task for later?

Reply:
This is for a follow on task later, it would mean introducing new e2e flags just for this one test, which I'm not sure is worth it.